source/documentation/set_up_a_live_worldpay_account.md

```diff
@@ -0,0 +1,164 @@
+
+If __Switch to Production__ is at the bottom of the left-hand menu, toggle it on.
+
+## Set up your profile
+
+To set up your profile, you need to copy credentials from your Worldpay account
+into your GOV.UK Pay account. 
+
+You will need both your [Worldpay
+account](https://secure.worldpay.com/sso/public/auth/login.html) and your
+[GOV.UK Pay account](https://selfservice.payments.service.gov.uk/login) open
+and signed in, in different web browser tabs.  
+
+1. Select __Profile__ in the left-hand menu in your Worldpay account.
+
+1. In your GOV.UK Pay browser tab, go to __Switch
+service__ to select the live service you want to set up. Select __Settings__,
+then __Account credentials__, then __Edit credentials__.
+
+1. In your Worldpay browser tab, go to the __Profile__ page, then select
+__Identification__. 
+
+1. Copy the Worldpay __Merchant Code__ into the GOV.UK Pay __Merchant ID__
+   field.
+
+1. Copy the Worldpay __Original XML Username__ into the GOV.UK Pay __Username__ field.
+
+1. Change the Worldpay XML password:
+
+    <br> select the pencil icon next to __XML Password__ </br>
+    <br> change the password and select __Add new Password__ </br>
+    <br> select __OK__ to go back to the __Profile__ page </br> 
+    <br> select the pencil icon again, then __Complete__ then __OK__ </br>
+    <br><br>
+
+1. Copy the Worldpay XML password into the GOV.UK Pay __Password__ field.
+
+1. In your GOV.UK Pay browser tab, select __Save credentials__ to go back to the
+__Account credentials__ page.
+
+1. In your Worldpay browser tab, go to the __Profile__ page. Go to the __Payment service__
+section and set __Capture delay (days)__ to __Off__.
+
+## Set up “Merchant Channel”
+
+In the left-hand menu of [your Worldpay
+account](https://secure.worldpay.com/sso/public/auth/login.html), select
+__Merchant Channel__. Go to the __http Protocol__ row of the __Merchant
+Channels (Production)__ category.
+
+You should see 4 groupings of settings. 
+
+For the first 2, __Merchant Channels (Production)__ and __Merchant Channels (Test)__: 
+
+1. Under __Active__, select __yes__.
+2. Set __Address__ to
+   `https://notifications.payments.service.gov.uk/v1/api/notifications/worldpay`.
+3. Set __Method__ to __POST__.
+4. Set __Client certificate__ to __no__.
+5. Under __Merchant Channels (Production)__ and __Active__, set the __email__ and __shopper
+   email__ protocols to __no__.
+
+For the next 2, also named __Merchant Channels (Production)__ and __Merchant Channels (Test)__: 
+
+Check the following in the __http__ row only:
+
+__SIGNED_FORM_RECEIVED__
+
+__SHOPPER_REDIRECTED__
+
+__SENT_FOR_AUTHORISATION__
+
+__AUTHORISED__
+
+__ERROR__
+
+__CANCELLED__
+
+__CAPTURED__
+
+__CAPTURE_FAILED__
+
+__SETTLED__
+
+__SETTLED_BY_MERCHANT__
+
+__CHARGED_BACK__
+
+__CHARGEBACK_REVERSED__
+
+__INFORMATION_REQUESTED__
+
+__INFORMATION_SUPPLIED__
+
+__EXPIRED__
+
+__SENT_FOR_REFUND__
+
+__REFUNDED__
+
+__REFUNDED_BY_MERCHANT__
+
+__REFUSED__
+
+__REFUSED_BY_BANK__
```

Comment:
@katiebates a small thing but in P-377 there is a discrepancy in the image between "TEST" and "PRODUCTION": would you mind verifying?